### DISCUSSION TITLE
Cant get interactsh-client to connect to self-hosted interactsh-server

### DISCUSSION THREAD

**Discussion by Anon0nyx** [projectdiscovery/interactsh#108] [Q&A] [answered]
Hello,

I purchased a domain through godaddy.com and configured it with the host names and nameservers provided in the configuration as well as a VPS server that the domain and host names are pointing to. Essentially, I have the two subdomains ns1 and ns2 for my domain and they are setup in the godaddy configuration as the nameservers for this domain. When I run the server it seems to work properly on its end, I.e. I am able to travel to my domain and access the interactsh webpage. But when I try to run the client with the command: /go/bin/interactsh-client -url mydomain.com it gives me the following error: [FTL] Could not create client: could not make register request: Post "oobtesting.mydomain.com/register": unsupported protocol scheme "" I am not in a place to afford burp collaborator through burp pro so I greatly appreciate this open source alternative and am excited to get it up an running. I appreciate any help and can provide any necessary files.

Note: I followed the instructions in the README file for the server exactly and did not do any major network configuring outside of that. I made sure that my domain through godaddy was pointing towards the public IP of the VPS I own through DigitalOcean, but that is about it for what I did outside of the README files. So if there is any further configuration I may need to do that; I followed all instruction steps online but am still receiving the same error. The server seems to be running and can be accessed through the browser, but the client does not connect to the server; the client can connect to the normal interactsh server though using the normal command: go/bin/interactsh-client.

Thank you!

**Comment by ehsandeep** [comment, marked as answer]
Hi, Correct command to connect self hosted server is `/go/bin/interactsh-client -server https://mydomain.com`

**Reply by Anon0nyx** [reply]
Ah, sometimes it is the simplest of things. Thank you for your help! It appears to be functional now.
